source/materials/OpticalMaterialProperties.cc

```diff
@@ -683,7 +658,7 @@ namespace opticalprops {
                                       G4double e_lifetime,
                                       G4double photoe_p)
   {
-    G4MaterialPropertiesTable* mpt      = new G4MaterialPropertiesTable();
+    G4MaterialPropertiesTable* mpt = new G4MaterialPropertiesTable();
 
     // PROPERTIES FROM XENON
     G4MaterialPropertiesTable* xenon_pt = opticalprops::GXe(pressure, temperature, sc_yield, e_lifetime);
```

Comment:
The variable `temperature` in `GXe` is defined but not used, do we want to keep it?

Reply:
I know, it is there, because we have written (but not used) a function that calculates the density, which depends also on the temperature. However, we are not using it, so I think we can remove the current dependence on the temperature and add it later, if needed.

Reply:
On a second thought, since the idea is to use the other function, it's probably better to stick to the current code and open a new PR to change the density function to the new version.